src/uwtools/tests/drivers/test_driver.py

```diff
@@ -3,9 +3,75 @@
 Tests for uwtools.drivers.driver module.
 """
 
+from importlib import resources
+from types import SimpleNamespace as ns
+from unittest.mock import patch
+
+import pytest
+from pytest import fixture
+
 from uwtools.drivers import driver
+from uwtools.drivers.driver import Driver
+
+
+@fixture
+def create_schema(tmp_path):
+    schema_file = tmp_path / "schema_file.jsonschema"
+    with open(schema_file, "w", encoding="utf-8") as f:
+        print("undefined_filter: '{{ 34 | not_a_filter }}'", file=f)
+    return schema_file
+
+
+@pytest.mark.usefixtures("create_schema")
+class Testinstance(Driver):
```

Comment:
nitpick: I find the word `instance` kind of odd in the name of a _class_ -- kind of a contradiction of terms. How about something like `TestDriver`, as it's a kind of `Driver`?

Reply:
Nitpick away, its great for me to be better in my work!